content/en/docs/community/release_checklist.md

```diff
@@ -397,21 +435,29 @@ always beneficial as it can be easy to miss something.
 
 When you are ready to go, hit `publish`.
 
-## 9. Update Docs Site
+## 9. Update Docs
 
 The [Helm website docs section](https://helm.sh/docs) lists the Helm versions
-for the docs. The minor and patch information needs to be updated on the site.
+for the docs. Major, minor, and patch versions need to be updated on the site.
 To do that create a pull request against the [helm-www
 repository](https://github.com/helm/helm-www). In the `config.toml` file find
-the proper `params.versions` section and updated the Helm version.
+the proper `params.versions` section and update the Helm version, like in this
+example of [updating the current
+version](https://github.com/helm/helm-www/pull/676/files).
+
+Close the [helm/helm milestone](https://github.com/helm/helm/milestones) for
+the release, if applicable.
```

Comment:
Not every patch release will have a milestone. But if a milestone exists, we do want to close it out after release.